src/App.tsx

```diff
@@ -14,11 +19,113 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.PureComponent<{}, State> {
+  state = {
+    goods: [...goodsFromServer],
+    onStart: true,
```

Comment:
strange variable name - onStart sounds like function, but it's boolean
```suggestion
    isStarted: true,
```